### ISSUE TITLE
Should ignore text in an cshtml attribute that is preceded by razor code

### ISSUE THREAD

**Issue by carlin-q-scott** [EWSoftware/VSSpellChecker#243]
The spell checker properly ignores HTML attribute text unless it contains razor template code, like in these examples:
```cshtml
<img src="@Model.BranchImage?fm=jpg&fl=progressive" />
```
fm, jpg and fl are flagged.

```cshtml
<div class="@(ViewBag?.CannotApply == true ? "collapse" : "") mt-2 mt-sm-0 p-md-0 text-center text-md-start mx-auto"></div>
```
sm, md and mx are flagged.

**Comment by EWSoftware** [comment]
Unless Visual Studio classifies it as code, all it knows is that it's a string.  If you can use a regular expression to match it in some way, you can use an [exclusion expression](http://ewsoftware.github.io/VSSpellChecker/html/6216eedb-6434-4cad-be06-576814e0b735.htm) to ignore it.
